data/pages-content/sponsor/information.md

```diff
@@ -7,46 +7,43 @@ subtitle:
 
 <!-- # Deadlines -->
 # Sponsor Information
```

Comment:
Do you think we could separate each item by hard deadline? What I mean is smth like this:
```
Monday, June 12 
* Sponsored Talk
* Sponsor Logo Submission
* Booth Customization
Monday, June 19
* Graphics Submission
...
```
That way I'm as a user immediately aware about 4 deadlines, all of them on Monday and all of them are hard deadlines

Reply:
@Kislovskiy does this look OK?  
![image](https://user-images.githubusercontent.com/6319134/234105305-fb1f94af-1b4f-4bbf-ab1b-d2dabdd250b2.png)

Reply:
implemented in https://github.com/EuroPython/website/pull/259/commits/346c8a8083779a7fd961e689b3f14e18b1f772e3 Let me know if it looks alright. Happy to improve!

Reply:
I like it:) Maybe we could add that the deadline includes the date